pc/src/menu.rs

```diff
@@ -1,26 +1,39 @@
-use std::vec;
-
 use crate::{update::init_level, Activity, GameState, Phase, LEVELS};
+use m3_models::{Key, ToPcGameEvent};
 
 use macroquad::{
 	hash,
 	prelude::*,
-	ui::{root_ui, widgets::Button, Skin}
+	ui::{root_ui, Skin}
 };
 
 const BUTTON_FONT_SIZE: u16 = 16;
 
-/*fn get_menu_skin() -> Skin {
+fn get_menu_skin() -> Skin {
 	{
```

Comment:
I don't think this extra brace that contains the entire function code serves any purpose